api/v1/swagger/paths/holdings.json

```diff
@@ -0,0 +1,352 @@
+{
+  "/holdings": {
+    "get": {
+      "x-mojo-to": "Holding#list",
+      "operationId": "listHoldings",
+      "tags": ["patrons", "holdings"],
+      "parameters": [
+        {
+          "name": "holding_id",
+          "in": "query",
+          "description": "Case insensitive starts-with search for internal holding identifier",
```

Comment:
This is a nice idea, but there are unintended consequences, such as trying to get a record with a specific low digits identifier, but getting all its bigger siblings,
eg.
  find 343 => receive [343, 3431, 3432, 3439, 34312145]
This would make it impossible to get only the specific holding_id